example_sp/djangosaml2_sp/saml2_sp/templates/amministrazione.html

```diff
@@ -1,7 +1,11 @@
 {% extends "base.html" %}
 {% load static %}
 {% block tab_active %} active{% endblock %}
+<<<<<<< HEAD
```

Comment:
here we have a merge conflict to be resolved

Reply:
here and in all the file

Reply:
Resolved in commit [40b7d2a](https://github.com/italia/Satosa-Saml2Spid/pull/93/commits/40b7d2a93a80fda0f814e31b495dc902291b7736)